game_functions/task_function.py

```diff
@@ -7,7 +7,7 @@ class task_functions:
     def __init__(self, win):
         self.trialClock=core.Clock()
         image_path = 'Images/Tasks/'
-        self.fixation=visual.TextStim(win, ori=0, font=u'Arial', pos=[0, 0], color=u'white',text=u'+')
+        self.fixation=visual.TextStim(win, ori=0, font=u'Arial', pos=[0, 0], height=50, color=u'white',text=u'+')
```

Comment:
When you develop a new feature, make a new branch for that feature from the current mainline branch.

When you then develop another new feature, you make that new branch again from the current mainline branch, not the existing feature branch.